initial_pose:
  position:
      x: -2.4384
      y:  2.4384
      z:  0.6604
  orientation:
      x:  0.000
      y:  0.000
      z:  0.928
      w:  0.372

opposite_ramp_top:
  position: 
    x: -2.4
    y: -2.4
    z:  0.6604
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.918187889516
    w: -0.396145174837

opposite_ramp_bottom:
  position: 
    x: -5.0
    y: -5.0
    z:  0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.396145174837
    w: 0.918187889516

simon_approach:
  position: 
    x: -4.000
    y: -5.615
    z:  0.000
  orientation: 
    x: 0.0
    y: 0.0
    z: 1.0
    w: 0.0
